modules/jaeger-deploy-production-es.adoc

```diff
@@ -0,0 +1,143 @@
+
+. Create a new project, for example `jaeger-system`.
+
+.. Navigate to *Home* -> *Projects*.
+
+.. Click *Create Project*.
+
+.. Enter `jaeger-system` in the *Name* field.
+
+.. Click *Create*.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. If necessary, select `jaeger-system` from the Project menu.  You may have to wait a few moments for the Operators to be copied to the new project.
+
+. Click the Jaeger Operator. On the *Overview* tab, under *Provided APIs*, the Operator provides a single link.
+
+. Under *Jaeger* click *Create Instance*.
+
+. On the *Create Jaeger* page, replace the default `all-in-one` yaml text with your production YAML configuration, for example:
+
++
+.Example jaeger-production.yaml file with Elasticsearch
+[source,yaml]
+----
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger-production
+  namespace:
+spec:
+  strategy: production
+  ingress:
+    security: oauth-proxy
+  storage:
+    type: elasticsearch
+    elasticsearch:
+      nodeCount: 1
+      redundancyPolicy: ZeroRedundancy
+    esIndexCleaner:
+      enabled: true
+      numberOfDays: 7
+      schedule: 55 23 * * *
+    esRollover:
+      image: 'jaegertracing/jaeger-es-rollover:1.13.1'
+      schedule: '*/30 * * * *'
+    options:
+      es:
+        server-urls: 'http://elasticsearch.default.svc:9200'
+    dependencies:
+      enabled: true
+      image: jaegertracing/spark-dependencies
+      schedule: 55 23 * * *
+----
++
+//TODO - find out if I should delete the dependencies?  No support for spark?
+
+. Click *Create* to create the Jaeger instance.
+
+
+. On the *Jaegers* page, click the name of the Jaeger instance, for example, `jaeger-prod-elasticsearch`.
+
+. On the *Jaeger Details* page, click the *Resources* tab.  Wait until all the pods have a status of "Running" before continuing.
+
+
+[id="jaeger-deploy-production-cli_{context}"]
+== Deploying Jaeger production from the CLI
+
+Follow this procedure to create an instance of Jaeger from the command line.
+
+.Prerequisites
+
+* An installed, verified {ProductName} Operator.
+* Access to the {product-title} Command-line Interface (CLI) commonly known as `oc`.
+* An account with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role.
++
+----
+$ oc login https://{HOSTNAME}:8443
+----
+
+. Create a new project named `jaeger-system`.
++
+----
+$ oc new-project jaeger-system
+----
+
+. Create a custom resource file named `jaeger-production.yaml` that contains the text of the example file in the previous procedure.
+
+. Run the following command to deploy Jaeger:
++
+----
+$ oc create -n jaeger-system -f jaeger-production.yaml
+----
++
+. Run the following command to watch the progress of the Pods during the installation process:
++
+----
+$ oc get pods -n jaeger-system -w
+----
++
+You should see output similar to the following:
+////
+TODO - test and confirm output
```

Comment:
:+1:  was going to say the single `jaeger` pod in the rendered version was wrong :)